### ISSUE TITLE
Translate "Cannot Be Blank" not working

### ISSUE THREAD

**Issue by Toni-hr98** [verbb/formie#1819]
### Question

When attempting to translate the error message "Cannot Be Blank" into Dutch, the translation remains in English.

Create a translation file for Dutch in the following directory: translations/nl/formie.php.

Add the necessary translations for the error message "Cannot Be Blank" as follows:

```
return [
    '"{attribute}" cannot be blank.' => '{attribute} mag niet leeg zijn.',
    "{attribute} cannot be blank." => "{attribute} mag niet leeg zijn.",
    "{label} cannot be blank." => "{label} mag niet leeg zijn.",
];
```

Expected Behavior:
After adding the translation entries and ensuring the correct file structure, the error message "Cannot Be Blank" should be displayed in Dutch ("mag niet leeg zijn")

Actual Behavior:
Despite providing the translation in the appropriate file, the error message remains in English, indicating that the translation is not being applied or recognized by the Formie plugin.

Additional Information:

Formie plugin version: dev-craft-5 as 3.0.0-beta.5
CRAFT CMS: 5.0.4
PHP: 8.3.2

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.6"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.7](https://github.com/verbb/formie/releases/tag/3.0.0-beta.7)
